components/job-orchestration/job_orchestration/executor/compress/fs_compression_task.py

```diff
@@ -71,6 +78,25 @@ def update_job_metadata_and_tags(db_cursor, job_id, table_prefix, tag_ids, archi
     )
 
 
+def upload_single_file_archive_to_s3(
+    archive_stats: Dict[str, Any],
+    archive_dir: pathlib.Path,
+    s3_config: S3Config,
+) -> Result:
+    archive_id = archive_stats["id"]
+
+    logger.info(f"Starting to upload archive {archive_id} to S3...")
+    src_file = archive_dir / archive_id
+    result = s3_put(s3_config, src_file, archive_id)
+    if not result.success:
+        logger.error(f"Failed to upload archive {archive_id}: {result.error}")
+        return result
+
+    logger.info(f"Finished uploading archive {archive_id} to S3...")
```

Comment:
```suggestion
    logger.info(f"Finished uploading archive {archive_id} to S3.")
```

We typically only use ellipses for operations that are in progress.